web_src/js/components/RepoActionView.vue

```diff
@@ -1,19 +1,19 @@
 <template>
   <div class="action-view-container">
     <div class="action-view-header">
-      <div class="action-info-summary gt-ac">
+      <div class="action-info-summary gt-ac gt-mt-5">
```

Comment:
```suggestion
      <div class="action-info-summary gt-ac gt-mt-4">
```

Seems a tad too much. Sorry to nitpick 😆